internal/database/migrations/04_create_organization.up.sql

```diff
@@ -19,3 +19,9 @@ CREATE TABLE `organization`
 );
 
 SET FOREIGN_KEY_CHECKS = 1;
+
+INSERT INTO `organization` (`name`, `created_by`, `last_updated_by`, `created_at`, `last_updated_at`, `is_active`)
+VALUES
+    ('Organization A', 1, 2, '2023-01-01 00:00:00', '2023-01-01 00:00:00', TRUE),
+    ('Organization B', 2, 3, '2023-02-01 00:00:00', '2023-02-01 00:00:00', TRUE),
+    ('Organization C', 3, 4, '2023-03-01 00:00:00', '2023-03-01 00:00:00', TRUE);
```

Comment:
We don't need to add organizations, I think this was done for testing purposes. 
So to account for testing the GET commands, you have to populate the organizations first using the Create command.

Reply:
Alright. This has been resolved.

Reply:
Cool! A [good reference](https://github.com/dembygenesis/theoverwatchtools.cli/blob/33dd61bccf6819341626cb14f1736f6bf80589f2/internal/persistence/persistors/mysqlstore/store_category_test.go#L377)